### PR TITLE
Added split/subaccount payment feature

### PR DESCRIPTION
**Pull request by GenerousDev**

### CODE REVIEW

**Comment by just1and0** [comment]
@GenerousDev I see you included your details in the .all-contributorsrc file. This should not be so. This file is auto-generated once your PR is approved by an admin.

Could you remove the changes made to this file so we can review your PR?

File: .all-contributorsrc [hunk 1]
```diff
@@ -152,6 +152,16 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
```
Comment (just1and0) [inline .all-contributorsrc:156]: @GenerousDev I see you included your details in the .all-contributorsrc file. This should not be so. This file is auto-generated once your PR is approved by an admin.

Could you remove the changes made to this file so we can review your PR?

**Review by just1and0 — CHANGES_REQUESTED** [review]

**Review by opmat — COMMENTED** [review]
Subaccount payment feature has already been added on version 4.0.4 from [commit 6ac4a80](https://github.com/just1and0/React-Native-Paystack-WebView/commit/6ac4a803259960ba76226ab82ba8993397c853da)

**Comment by just1and0** [comment]
done on v4

**Pull request closed without merging**